config/Development.toml

```
@@ -103,6 +103,9 @@ base_url = "https://apis.sandbox.globalpay.com/ucp/"
 [connectors.worldline]
 base_url = "https://eu.sandbox.api-ingenico.com/"
 
+[connectors.bluesnap]
+base_url = "https://sandbox.bluesnap.com"
```

It would be better if you put `/` in the end of the url
```suggestion
base_url = "https://sandbox.bluesnap.com/"
```